src/servers/http_server_v2.cc

```diff
@@ -271,12 +271,13 @@ class HTTPAPIServerV2 : public HTTPServerV2Impl {
       }
     }
 
+    void SetRequestId(uint64_t request_id) { request_id_ = request_id; }
```

Comment:
The id is available in the InferenceRequest object, does it need to be maintained here as well?